api/apps/api/test/access-control/project-acl-delete.e2e-spec.ts

```diff
@@ -0,0 +1,38 @@
+import { FixtureType } from '@marxan/utils/tests/fixture-type';
+import { getFixtures } from './project-acl.fixtures';
+
+let fixtures: FixtureType<typeof getFixtures>;
+
+beforeEach(async () => {
+  fixtures = await getFixtures();
+});
+afterEach(async () => {
+  await fixtures?.userRoleCleanup();
```

Comment:
Do we have any particular reason why `userRoleCleanup` cannot be a part of `cleanup`? 

The idea of `cleanup` is to remove all "noise" from the test itself

Reply:
I did it to separate it from the regular cleanup and only apply it when a project is created, so it does not run everytime I apply a cleanup.

Now that I think of, maybe it is better if I include it in cleanup but I remove cleanups from the before/after GET tests that use the basic seeded data. 🤔

Reply:
you can always `push` to `cleanups` within given step implementation, how about this?

Reply:
You mean like including it in the `GivenProjectWasCreated` for example?

Reply:
@rubvalave yes, exactly.